live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -323,3 +323,33 @@
   with_items:
     - { key: 'kernel.core_pattern', value: '/var/crash/core.%e.%p.%t' }
     - { key: 'vm.overcommit_memory', value: '1' }
+
+#
+# Add the apt repos that have the debug versions of packages, so that we
+# can download a debug kernel to use with crash.
+#
+- apt_repository:
+    repo: "{{ item }}"
+  with_items:
+    - deb http://ddebs.ubuntu.com bionic main restricted universe multiverse
```

Comment:
If we're going to be adding `ddebs.ubuntu.com` to our build, we might as well start adding some useful userland debug symbols in addition to the kernel debug symbols that you are adding in this change (e.g., debug symbols for the C library, Java and Python runtimes, etc.). I don't think this needs to be done as part of your current change, but your change definitely enables it.